<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSQL</class>
 <widget class="QMainWindow" name="QSQL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QSQL</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionInsert"/>
   <addaction name="actionEdit"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionBonus"/>
   <addaction name="separator"/>
   <addaction name="actionSQLTest"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/open3.bmp</normaloff>:/images/open3.bmp</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/306.bmp</normaloff>:/images/306.bmp</iconset>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/812.bmp</normaloff>:/images/812.bmp</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/delete1.bmp</normaloff>:/images/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionBonus">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/up.bmp</normaloff>:/images/up.bmp</iconset>
   </property>
   <property name="text">
    <string>Bonus</string>
   </property>
  </action>
  <action name="actionSQLTest">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/828.bmp</normaloff>:/images/828.bmp</iconset>
   </property>
   <property name="text">
    <string>SQLTest</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/exit.bmp</normaloff>:/images/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>QSQL</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
